src/modal/modal.component.ts

```diff
@@ -335,6 +336,14 @@ export class ModalDirective implements AfterViewInit, OnDestroy {
   // }
   // }
 
+  protected focusOtherModal() {
+    const otherOpenedModals = this._element.nativeElement.parentElement.querySelectorAll('.in[bsModal]');
```

Comment:
this is so bad ;(
but other ideas how to connect modals  is harder to implement